python/res/fm/ecl/script.py

```diff
@@ -0,0 +1,22 @@
+from argparse import ArgumentParser
+from .ecl_run import EclRun
+
+
+
+
+def run(config, argv):
+    parser = ArgumentParser()
+    parser.add_argument("ecl_case")
+    parser.add_argument("--version", dest="version", type=str)
```

Comment:
`default=Keys.default`?

Reply:
I thought we should not expose the `Keys` class - current code will pass `None` for version as default; and that should be picked up as good as the `Keys.default` string. The main purpose of the `Keys.default` string is to support the default in the old style FORWARD_MODEL file.